examples/facebook.amp.html

```diff
@@ -25,6 +25,12 @@ <h2>Facebook</h2>
 
 <h1>More Posts</h1>
 
+<amp-facebook width=552 height=303
```

Comment:
I don't see any changes to `getFacebookSdk()` in `facebook.js` , a bit confused on how this is working, missed a commit?

Reply:
Added the missing file. Sorry about that!